app/web/components/ImageInput/ImageInput.tsx

```diff
@@ -77,7 +77,8 @@ const useStyles = makeStyles((theme) => ({
 
 interface ImageInputProps {
   className?: string;
-  control: Control;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  control: Control<any>;
```

Comment:
I wonder if making this extends from `FieldValues` and let it passed in as a generic param would work instead to avoid the `any`? Appreciate the type golf with react-hook-form can get hairy though...

Reply:
Not gonna lie, I spent full DAYS on these types during the Thanksgiving break 😅.

I did try `FieldValues`. Every time I get one component's types working it breaks the parent components. Or I get both props passing TS checks and the useController types get mad.

I cannot figure it out and I spent so, so much time on it before I decided it wasn't worth that much effort.

Totally happy if you wanna push a commit to this PR if something obvious stands out to you though! Or can do after.